include/common/gazebo_node.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+// Gazebo includes
+#include <gazebo/transport/transport.hh>
+#include <gazebo/gazebo_client.hh>
+
+using namespace gazebo::transport;
```

Comment:
I don't think this belongs in common and it definitely should be in a namespace - personally I'd say ``BoBRobotics::Gazebo::getNode`` and ``BoBRobotics::Gazebo::shutdown`` would be better names

Reply:
:+1: